### PR TITLE
docs: update english thousand separator example

### PR DESCRIPTION
**Pull request by willfarrell**
Closes: #234

### CODE REVIEW

File: guide/variables.md [hunk 1]
```diff
@@ -43,14 +43,14 @@ time-elapsed = Time elapsed: { $duration }s.
 ```
 
 For the `$duration` variable value of `12345.678`, the following message will be
-displayed in the product, assuming the language is set to British or American
+displayed in the product, assuming the language is set to British, Canadian, or American
```
Comment (eemeli) [inline guide/variables.md:46]:
```suggestion
displayed in the product, assuming the language is set to British or American
```

Given the discussion in #234, let's avoid mentioning Canada here. The specific cases we're comparing don't really matter, as the overall point is that It's Complicated, and that's actually easier to make by leaving out Canada, which itself Is Complicated.

File: guide/variables.md [hunk 2]
```diff
@@ -43,14 +43,14 @@ time-elapsed = Time elapsed: { $duration }s.
...
 
 ```
 Time elapsed: 12,345.678s.
 ```
 
-In Canadian English, however, the result would be:
+In South Africian English, however, the result would be:
 
 ```
 Time elapsed: 12 345.678s.
```
Comment (eemeli) [inline guide/variables.md:56]:
```suggestion
Time elapsed: 12 345,678s.
```

South Africa uses comma as the decimal separator.

**Review by eemeli — CHANGES_REQUESTED** [review]

**Comment by willfarrell** [comment]
Thanks for the review, should be good to go now.

**Review by eemeli — APPROVED** [review]

**Comment by SirNickolas** [comment]
African, not Africian.

File: guide/variables.md [hunk 3]
```diff
@@ -50,10 +50,10 @@ English. Note the use of comma as the thousands separator.
 Time elapsed: 12,345.678s.
 ```
 
-In Canadian English, however, the result would be:
+In South Africian English, however, the result would be:
```
Comment (eemeli) [inline guide/variables.md:53]:
```suggestion
In South African English, however, the result would be:
```

**Comment by eemeli** [comment]
@willfarrell Thank you for noticing and fixing this!

Also @SirNickolas thank you for noticing the typo!

**Pull request merged**